### ISSUE TITLE
Righthand Sidebar Doesn't Show All Headers.

### ISSUE THREAD

**Issue by Auratelience** [notaapp/talk#325]
I have a file with 150 level 2 headers (plus one level-one header as a title) and the sidebar only shows the first 64.

**Comment by astoilkov** [comment]
Yeah. We did that on purpose. We still haven't optimized the performance of the Right Sidebar and adding all the headings can cause some slowdown in the editor experience on some computers. We will improve on that.

**Comment by Auratelience** [comment]
Ah ok. It’s not incredibly important of it can’t be done.

**Comment by Auratelience** [comment]
Maybe if there is an experimental/beta setting to enable it, even at the risk of performance could be a good in-between?
